### DISCUSSION TITLE
Any way to save logs of log-level to a file every time postgrest started?

### DISCUSSION THREAD

**Discussion by lokize** [PostgREST/postgrest#2360] [Q&A] [answered]
Hi is there any way to save logs of postgrest to a file?

=========edit=========

i using >> SomeFile.txt to running command but i think maybe its not best way to make that...

**Comment by steve-chavez** [comment, marked as answer]
Hey @lokize,

Not directly but you can send the logs to files with systemd as described [here](https://unix.stackexchange.com/a/479669/278242).

Related:
- https://postgrest.org/en/stable/admin.html#logging
- https://postgrest.org/en/stable/admin.html#daemonizing
